ExtentReportsDotNetCLI/ExtentReportsDotNetCLI/Parser/NUnitParser.cs

```diff
@@ -70,6 +71,11 @@ public void ParseTestRunnerOutput(string resultsFile)
 
                     AssignStatusAndMessage(tc, node);
                     AssignTags(tc, node);
+
+                    DateTime.TryParse(tc.Attribute("start-time").Value, out timeStampParsed);
```

Comment:
There needs to be a null check here. Note that the `start-time` and 'end-time' attributes are not available prior to version 3 of NUnit.